### ISSUE TITLE
I am getting "is not a mailbox" showing when I change accounts.

### ISSUE THREAD

**Issue by DrDonkey** [LukeSmithxyz/mutt-wizard#32]
i1-13 works fine but i4-i7 doesn't work for me. I have 7 accounts set up in neomutt using the wizard.
Tried looking in configs but didn't find anything useful. And the keys are also defined in .personal.muttrc.

**Comment by LukeSmithxyz** [comment]
Is there anything in common between your accounts 4-7? It's possible they may've come from an email provider with strange box naming conventions and thus might not have been detected correctly? Or are they non-English?

Could you post one of the accounts/NAME.muttrc generated?

**Comment by DrDonkey** [comment]
Weird thing actually, there was 7 accounts listed in neomutt but there were only 6 in accounts folder. So I checked carefully and found that I added one account twice. So I went to personal.muttrc, deleted one line that wasn't present in the accounts folder. But after that when I tried offlineimap, I got error saying `" " is not a valid account` or something similar. So I just removed all from the mutt wizard sh, and added them. And now its all fine.

Maybe the problem was the account just conflicted with each other or something, didn't know. But it works now.
